src/Pyz/Yves/ShopUi/Theme/default/components/molecules/suggest-search/suggest-search.ts

```diff
@@ -0,0 +1,115 @@
+import SuggestSearchCore from 'ShopUi/components/molecules/suggest-search/suggest-search';
+import {
+    EVENT_HIDE_OVERLAY,
+    EVENT_SHOW_OVERLAY,
+    OverlayEventDetail,
+} from 'ShopUi/components/molecules/main-overlay/main-overlay';
+
+export default class SuggestSearch extends SuggestSearchCore {
+    protected overlay: HTMLElement;
+    protected wrapper: HTMLElement;
+    protected openTrigger: HTMLElement;
+    protected closeTrigger: HTMLElement;
+    protected eventShowOverlay: CustomEvent<OverlayEventDetail>;
+    protected eventHideOverlay: CustomEvent<OverlayEventDetail>;
+
+    protected init(): void {
+        this.overlay = <HTMLElement>document.getElementsByClassName(this.overlayClassName)[0];
+        this.wrapper = <HTMLElement>document.getElementsByClassName(this.wrapperClassName)[0];
+        this.openTrigger = <HTMLElement>document.getElementsByClassName(this.openClassName)[0];
+        this.closeTrigger = <HTMLElement>document.getElementsByClassName(this.closeClassName)[0];
+
+        super.readyCallback();
+    }
+
+    protected mapEvents(): void {
+        super.mapEvents();
+
+        this.openTrigger.addEventListener('click', () => this.toggleOverlay(true));
```

Comment:
Looks strange that we show the overlay on the click and on `showSugestions` method...